index.bs

```diff
@@ -544,8 +544,8 @@ dictionary XRRenderStateInit {
   readonly attribute double depthNear;
   readonly attribute double depthFar;
   readonly attribute double? inlineVerticalFieldOfView;
-  readonly attribute XRLayer? baseLayer;
-  readonly attribute XRPresentationContext? outputContext;
+  [SameObject] readonly attribute XRLayer? baseLayer;
+  [SameObject] readonly attribute XRPresentationContext? outputContext;
```

Comment:
These two attributes are the only ones I question in this change, because it implies that each time the render state changes the session attribute will be a new object. That's actually not a bad assumption to make, and it's probably preferable to having `XRRenderState` be a live object, but I'd like to get that clearly defined one way or the other in the spec prior to adding a dependency on that behavior here.

Reply:
Oh, hm, that's fair. I'll remove that and file an issue